src/main/java/teammates/common/datatransfer/questions/FeedbackMcqQuestionDetails.java

```diff
@@ -242,11 +248,24 @@ public String getQuestionWithoutExistingResponseSubmissionFormHtml(
             break;
         case STUDENTS:
             List<StudentAttributes> studentList = StudentsLogic.inst().getStudentsForCourse(courseId);
-
             for (StudentAttributes student : studentList) {
                 optionList.add(student.name + " (" + student.team + ")");
             }
 
+            optionList.sort(null);
+            break;
+
+        case STUDENTS_EXCLUDING_SELF:
```

Comment:
This block looks almost identical to the `STUDENTS` one. Why not use the same block and only do extra logical steps when the option is `STUDENTS_EXCLUDING_SELF`?

Reply:
```
case STUDENTS:
    //fallthrough
case STUDENTS_EXCLUDING_SELF:
    List<StudentAttributes> studentList = StudentsLogic.inst().getStudentsForCourse(courseId);

    if(generateOptionsFor == FeedbackParticipantType.STUDENTS_EXCLUDING_SELF) {
        studentList.removeIf(studentInList -> studentInList.email.equals(studentDoingQuestion.email));
    }

    for (StudentAttributes student : studentList) {
        optionList.add(student.name + " (" + student.team + ")");
    }

    optionList.sort(null);
    break;
```

Is this okay?